io/plugins/eu.esdihumboldt.hale.io.jdbc.test/src/eu/esdihumboldt/hale/io/jdbc/test/DockerConfig.java

```diff
@@ -0,0 +1,92 @@
+package eu.esdihumboldt.hale.io.jdbc.test;
+
+import java.io.File;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+
+public class DockerConfig {
+	public static final String DOCKER_CONF = "docker.conf";
```

Comment:
I would prefer to not use a filename like `docker.conf` - this is too prone to conflicts with something else related to Docker.
I would suggest something that is more specific, like `hale-docker.conf` or something like that. An option could also be sticking to `overcast.conf` if we keep a format that is compatible to their configuration.